(no ticket): [external] Stencil, clarify that similar_by_views property is inactive (#13)
bigcommerce/docs: docs/stencil-docs/reference-docs/front-matter-reference.mdx

      limit: 10        # limits similar products by views to 10
```

<Callout type="info">

> NOTE: similar_by_views is currently not enabled on the platform.

Reviewer: ```suggestion
NOTE: The `similar_by_views` property is not currently active.
```

> NOTE: The `similar_by_views` property is currently not enabled on the platform.

Reviewer: ```suggestion
NOTE: The `similar_by_views` property is not currently active.
```

|`videos`|If `product.videos` is not defined, you will not return videos. If you define `product.videos`, the default behavior is to return all videos. If you define `product.videos.limit`, this sets the maximum number of videos returned.|
|`images`|If `product.images` is not defined, you will not return images. If `product.images` is defined, you must also define `product.images.limit`, which throttles the number of images returned. The maximum allowable value for this parameter is five images.|
|`reviews`|Boolean indicating whether to display product reviews. If `product.reviews` is present and is not explicitly set to `false`, reviews will appear. If not defined, defaults to 10 reviews. When filtering/limiting reviews, the default sorting is by review id, from lowest to highest. If a product has over 250 reviews, you can fetch the rest using the GraphQL Storefront API. See the example GraphQL query below.|
|`related_products`|Displays products that are related by name. If `limit` absent or undefined, the default behavior is to display all related products. Inserting `limit` with no integer will display 0 products.|

> |`similar_by_views`|Displays products similar to those displayed in the current page context. If `limit` absent or undefined, the default is to display four products. (Currently disabled on the platform)|

Reviewer: ```suggestion
|`similar_by_views`|Displays products similar to those displayed in the current page context. If `limit` absent or undefined, the default is to display four products. Currently disabled on the platform. |
```

  }
}"
```

<Callout type="info">

> NOTE: similar_by_views is currently not enabled on the platform.

Reviewer: ```suggestion
NOTE: The `similar_by_views` property is not currently active.
```

> </Callout>`

Reviewer: ```suggestion
</Callout>
```
